website/src/content/getting-started/what-is-a-custom-application.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: What is a custom application?
+---
+
+<Subtitle>
+
+A custom application is a UI web application for the Merchant Center, designed to enable a project in the [commercetools platform](https://commercetools.com/) to tailor specific needs of a customer.
+
+</Subtitle>
+
+# About the Merchant Center
+
+The Merchant Center is an **administration tool to help customers manage the data** of their projects in the [commercetools platform](https://commercetools.com/).
```

Comment:
Isn't it more data we allow managing (orders et al) or is this the general term we use?

Reply:
That's what it says in the mc docs: https://docs.commercetools.com/merchant-center/